clarifai/client/input.py

```diff
@@ -499,38 +504,64 @@ def get_bbox_proto(input_id: str, label: str, bbox: List, label_id: str = None)
     """
     if not isinstance(bbox, list):
       raise UserError("must be a list of bbox cooridnates")
-    input_annot_proto = resources_pb2.Annotation(
-        input_id=input_id,
-        data=resources_pb2.Data(regions=[
-            resources_pb2.Region(
-                region_info=resources_pb2.RegionInfo(bounding_box=resources_pb2.BoundingBox(
-                    # bbox ordering: [xmin, ymin, xmax, ymax]
-                    # top_row must be less than bottom row
-                    # left_col must be less than right col
-                    top_row=bbox[1],  #y_min
-                    left_col=bbox[0],  #x_min
-                    bottom_row=bbox[3],  #y_max
-                    right_col=bbox[2]  #x_max
-                )),
-                data=resources_pb2.Data(concepts=[
-                    resources_pb2.Concept(
-                        id=f"id-{''.join(label.split(' '))}", name=label, value=1.)
-                    if not label_id else resources_pb2.Concept(id=label_id, name=label, value=1.)
-                ]))
-        ]))
+    if annot_id:
+      input_annot_proto = resources_pb2.Annotation(
+          id=annot_id,
```

Comment:
Is `annot_id` being None giving error? If thats the case, we can assign `resouce_pb2.Data` in a temp variable and reuse it in the if else condition.

Reply:
Done